source/documentation/strategy-and-principles/re-strategy.html.md

```diff
@@ -1,40 +1,40 @@
 # RE Strategy
 
-This is the Reliability Engineering Strategy for 2019-2020. The main text of the strategy is in bold with further explanation and reasoning underneath.
+<b>This is the Reliability Engineering Strategy for 2019-2020 which aims to set a direction and provide guidance for RE to meet its objectives for convergence and efficiency.</b>
```

Comment:
Why is there HTML in Markdown? 😱

Reply:
Haha. Because it's the tidiest way I could find of getting several paragraphs at a time smaller and in italics without splatting asterisks absolutely everywhere. And having done it for that, I felt bad about NOT doing it in the one other place.